microsoft-edge/web-platform/ad-selection-api.md

```diff
@@ -0,0 +1,300 @@
+
+   The `attestations.json` JSON file must be published within **30 days** of receiving the OT token.
+
+
+<!-- ------------------------------ -->
+#### Example attestations.json file
+
+The `attestations.json` file for a domain must conform to the following standards below.  The file must include the generated OT token for the top-level domain, as well as a list of APIs (features that are associated with the Ad Services API).
+
+The following is an example `attestations.json` file:
+
+```json
+{
+    "ad_selection_api_attestations": [
+        {
+            "attestation_parser_version": "2",
+            "attestation_version": "2",
+            "privacy_policy": [
+                "https://contoso.example/privacy/"
+            ],
+            "ownership_token": "<Your Unique Token Here>",
+            "enrollment_site": "https://contoso.example/",
+            "platform_attestations": [
+                {
+                    "platform": "edge",
+                    "attestations": {
+                        "attribution_reporting_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        },
+                        "shared_storage_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        },
+                        "private_aggregation_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        },
+                        "ad_selection_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        }
+                    }
+                },
+                {
+                    "platform": "android",
+                    "attestations": {}
+                }
+            ]
+        }
+    ]
+}
+```
+
+
+<!-- ---------- -->
+###### Important fields and values
+
+* `"ownership_token":` is the OT token that was generated when registering this domain for the **Ad Selection API** origin trial.
+
+* In the `"platform_attestations":` section, `"platform":` must be `"edge"` or `"android"`.
+
+* In the `"platform_attestations":` section, valid members of `"attestations":` are the following:
+   * `"attribution_reporting_api":` - Part of the Ad Selection API, for attribution reporting.
+   * `"shared_storage_api":` - Part of the Ad Selection API, for shared storage.
+   * `"private_aggregation_api":` - Part of the Ad Selection API, for private aggregation.
+   * `"ad_selection_api":`
+   <!-- todo: is this list complete?  eg is there "fenced_frames_api": {} ?  per list in flag desc -->
+
+   Each of these `"attestations":` entries must have a single field, `"ServiceNotUsedForIdentifyingUserAcrossSites":`, with either a `true` or `false` value.  `true` means that this service is not used for identifying the user across sites.  `false` means that this service is used for identifying the user across sites.
```

Comment:
Is this indent for 'Each of these `"attestations":` entries must have a single field...'  is okay? 

![image](https://github.com/user-attachments/assets/14a831eb-9c42-4412-8f3a-1a1c69eb96a6)

Reply:
i agree that it looks possibly incorrect, even though it is technically indented the correct amount.  will consider format alternatives.

Reply:
result:
![image](https://github.com/user-attachments/assets/286a79a2-f0c3-4e98-a240-0d434038bc90)